bikeshare_app/views.py

```diff
@@ -107,6 +107,13 @@ def register():
         # Allow user through if check passed, else log attempt and return home
         return render_template('register.html')
 
+@app.route('/checkin', methods=['GET'])
```

Comment:
Hey bud `methods` is by default equal to `['GET']` there's no reason to specify it here unless someone else has a particularly strong feeling